tsfile/src/main/java/org/apache/iotdb/tsfile/read/common/TimeColumn.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iotdb.tsfile.read.common;
+
+
+public class TimeColumn {
+
+  private long[] times;
+
+  private int size;
+
+  private int cur;
+
+
+  public TimeColumn(int initSize) {
+    times = new long[initSize];
+  }
+
+
+  public TimeColumn(long[] times) {
```

Comment:
This constructor is not used?

Reply:
not used , it can used for 0 copy batchdata's timeRet.